include/tvm/relay/error_reporter.h

```diff
@@ -92,6 +92,10 @@ class ErrorReporter {
    */
   [[noreturn]] void RenderErrors(const Module& module, bool use_color = true);
 
+  inline bool AnyErrors() {
```

Comment:
I think `inline` is not needed for in-class members as compilers should do that by default.